doc/source/jupyterhub/customizing/user-management.md

```diff
@@ -7,19 +7,26 @@ permissions on JupyterHub.
 
 ## Culling user pods
 
-JupyterHub uses the [jupyterhub-idle-culler](https://github.com/jupyterhub/jupyterhub-idle-culler) to automatically delete any user pods that have no activity
-for a period of time. This helps free up computational resources and keeps
-costs down if you are using an autoscaling cluster.
-When these users navigate back to your JupyterHub, they will
-have to start their server again, and the state of their previous session
-(variables they've created, any in-memory data, etc)
-will be lost. This is known as _culling_.
+When users work with a JupyterHub deployment, will they reliably shut down their
+servers manually when finished? Probably not. Due to this, it can be good to
+have a system to shut down servers that are inactive after a configurable
+duration of inactivity or a maximum amount of time. JupyterHub uses the
+[jupyterhub-idle-culler](https://github.com/jupyterhub/jupyterhub-idle-culler)
+to do this.
+
+When a user server has been shut down, the user will have to start their server
+again on next visit. The in-memory state of their previous session will have
+been lost, but opened notebooks are saved regularly into a `.ipynb_checkpoints`
+folder and will typically not cause work to be lost.
 
 ```{note}
-In JupyterHub, "inactivity" is defined as no response from the user's
-browser. The `jupyterhub-idle-culler` running on the `hub` pod constantly pings the user's JupyterHub browser session via the JupyterHub API
-to check whether it is open. This means that leaving the computer running
-with the JupyterHub window open will **not** be treated as inactivity. However, leaving a process or notebook running and closing the web socket (by closing the browser) **will** be treated as inactivity.
+For more details on how the `jupyterhub-idle-culler` works and additional
+configurations you may want to set on the user servers, see the [How it works
+documentation](https://github.com/jupyterhub/jupyterhub-idle-culler#how-it-works).
+
+If you want to configure the culling of kernels that can help stop long running
+code on the user servers, it can be useful to use
+[`singleuser.extraFiles`](schema_singleuser.extraFiles).
```

Comment:
I appreciate that this is referencing your solution [here](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/2244#issuecomment-865085937) for providing the `jupyter_notebook_config.py` file, which is very useful. I think as written, though, it is hard to understand how `singleuser.extraFiles` can help with culling. 

I might rewrite this as:
```
If you want to configure the culling of kernels using, for example, a kernel manager that can help stop long running
code on the user servers, it can be useful to use
[`singleuser.extraFiles`](schema_singleuser.extraFiles) to pass a `jupyter_notebook_config.py` file to `singleUser` servers with those configuration values like those documented in the [jupyterhub-idle-culler docs[(https://github.com/jupyterhub/jupyterhub-idle-culler#how-it-works) 
```

Reply:
I tried addressing this in 6b8cef8! Thanks for your input!